Plots/Satellites/NCL_sat_1.py

```diff
@@ -0,0 +1,191 @@
+"""
+NCL_sat_1.py
+===============
+This script illustrates the following concepts:
+   - Creating an orthographic projection
+   - Drawing line contours over a satellite map
+   - Manually labeling contours
+   - Transforming coordinates
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/sat_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/sat_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import xarray as xr
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import numpy as np
+
+import geocat.datafiles as gdf
+import geocat.viz.util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/slp.1963.nc"), decode_times=False)
+
+# Get data from the 24th timestep
+U = ds.slp[24, :, :]
+
+# Translate short values to float values
+U = U.astype('float64')
+
+# Convert Pa to hPa data
+U = U*0.01
+
+# Fix the artifact of not-shown-data around 0 and 360-degree longitudes
+wrap_U = gvutil.xr_add_cyclic_longitudes(U, "lon")
+
+###############################################################################
+
+def findCoordPressureData(coordarr, lat, lon):
+    # Finds pressure at coordinate given lat, lon, and an array of all the coordinates with data
+    # Lat/lon is not *real* lat/lon.... based on scale given by xarray data [90, -90] and [0, 360]
+    # rather than [-90, 90] and [-180, 180]
+    for x in range(len(coordarr)):
+        for y in range(len(coordarr[x])):
+            if coordarr[x][y][0] == lat and coordarr[x][y][1] == lon:
+                return U.data[x][y]
+
+def makeCoordArr():
+    # Returns an array of (lat, lon) coord tuples with the same dimensions as the pressure data
+    coordarr = []
+    for x in np.array(U.lat):
+        temparr = []
+        for y in np.array(U.lon):
+            temparr.append((x,y))
```

Comment:
Longitudes are the x coordinates and latitudes are the y coordinates for cartopy. I suggest changing this to reflect that fact.